16 - Mouse Move Shadow/index-START.html

```diff
@@ -1,40 +1,64 @@
 <!DOCTYPE html>
 <html lang="en">
-<head>
-  <meta charset="UTF-8">
-  <title>Mouse Shadow</title>
-</head>
-<body>
-
-  <div class="hero">
-    <h1 contenteditable>🔥WOAH!</h1>
-  </div>
-
-  <style>
-  html {
-    color: black;
-    font-family: sans-serif;
-  }
-
-  body {
-    margin: 0;
-  }
-
-  .hero {
-    min-height: 100vh;
-    display: flex;
-    justify-content: center;
-    align-items: center;
-    color: black;
-  }
-
-  h1 {
-    text-shadow: 10px 10px 0 rgba(0,0,0,1);
-    font-size: 100px;
-  }
-  </style>
-
-<script>
-</script>
-</body>
+  <head>
+    <meta charset="UTF-8" />
+    <title>Mouse Shadow</title>
+  </head>
+  <body>
+    <div class="hero">
+      <h1 contenteditable>🔥WOAH!</h1>
+    </div>
+
+    <style>
+      html {
+        color: black;
+        font-family: sans-serif;
+      }
+
+      body {
+        margin: 0;
+      }
+
+      .hero {
+        min-height: 100vh;
+        display: flex;
+        justify-content: center;
+        align-items: center;
+        color: black;
+      }
+
+      h1 {
+        text-shadow: 10px 10px 0 rgba(0, 0, 0, 1);
+        font-size: 100px;
+      }
+    </style>
+
+    <script>
+      const hero = document.querySelector(".hero");
+      const woah = document.querySelector(".hero h1");
+      const distance = 500;
+
+      const handleMouseMove = (e) => {
+        let mouseX = e.offsetX;
+        let mouseY = e.offsetY;
+
+        if (e.target !== hero) {
+          mouseX = e.offsetX + e.target.offsetLeft;
+          mouseY = e.offsetY + e.target.offsetTop;
+        }
```

Comment:
@ct-aa, we may use the `mouseX` and `mouseY` here instead of `e.offsetX` & `e.offsetY` since you already have those in line 42 and 43.